### ISSUE TITLE
Navbar responsiveness

### ISSUE THREAD

**Issue by Guli93** [akshitagupta15june/Moksh#335]
navbar is not responsive at all 

please assign this issue to me under Gssoc'23

**Comment by sirajlekhak** [comment]
I am eager to contribute as a valuable team member and demonstrate my skills in GSSoC'23. If you would kindly assign me an issue, I would be delighted to work on it and showcase my abilities effectively within the program.

**Comment by karthiknadar1204** [comment]
hii @Guli93 This issue has already been assigned and is being reviewed right now, Please close this issue for now.
Explore other issues to contribute to.
Please check if an issue is already raised before raising a similar issue, we will review the issue and assign them accordingly.
Thank you

**Comment by sabhisharma-ise** [comment]
@karthiknadar1204 Please check the PR #321 regarding this issue. Already fixed the Issue!

**Comment by karthiknadar1204** [comment]
Yes, the pr will be merged soon by @akshitagupta15june when she is free.
Thank you
